src/main/java/opendota/Parse.java

```diff
@@ -552,7 +552,9 @@ private Item getHeroItem(Context ctx, Entity eHero, int idx) throws UnknownItemF
         Item item = new Item();
         item.id = itemName;
         item.num_charges = eItem.getProperty("m_iCurrentCharges");
+        if(item.num_charges == 0) item.num_charges = null;
```

Comment:
maybe just wrap the assignment in if:
```
int numCharges = eItem.getProperty("m_iCurrentCharges");
if (numCharges != 0) {
  item.num_charges = numCharges;
}
```

(I also prefer to use braces even for single line blocks